packages/client/src/components/CommentBox.vue

```diff
@@ -206,11 +206,12 @@
           class="wl-emoji-popup"
           :class="{ display: showEmoji }"
         >
-          <template
-            v-for="(emojiItem, index) in emoji.tabs"
-            :key="emojiItem.name"
-          >
-            <div v-if="index === emojiTabIndex" class="wl-tab-wrapper">
+          <template v-for="(emojiItem, index) in emoji.tabs">
+            <div
+              v-if="index === emojiTabIndex"
+              :key="emojiItem.name"
+              class="wl-tab-wrapper"
+            >
```

Comment:
![image](https://user-images.githubusercontent.com/33315834/189706584-5cc8ec2a-b78e-42e9-b7b7-ca8e6455986a.png)

Reply:
I change it because of lint error(https://eslint.vuejs.org/rules/no-v-for-template-key.html), so which one is right?

<img width="1215" alt="image" src="https://user-images.githubusercontent.com/424491/189925988-af1c4d61-4abe-47ea-adf7-6a30d4d4a4f8.png">

Reply:
> This rule is targeted at Vue.js 2.x. If you are using Vue.js 3.x, enable the [vue/no-v-for-template-key-on-child](https://eslint.vuejs.org/rules/no-v-for-template-key-on-child.html) rule instead. Don't enable both rules together; they are conflicting.

Maybe we should modify eslint config to remove `vue/no-v-for-template-key` rule?

Reply:
Update: It seems lead in by my vscode plugin `Vetur` ... All runs well when I forbidden it.